.github/workflows/tests.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           ini-values: display_errors=off, log_errors=on
+          extensions: :xdebug
```

Comment:
adding `xdebug` is not relevant for the scope of this PR, is it?

Reply:
details are in the commit message.